gui/mountPrimaryWindow.py

```diff
@@ -18,6 +18,8 @@
 
 bucketOptions = ['s3storage', 'azstorage']
 mountTargetComponent = 3
+defaultFuseDir = '\Cloudfuse'
```

Comment:
Since this is already defined in common_qt_functions.py, is it possible to define this in just one place, with other shared constants?

Reply:
Agreed